config/initializers/delayed_job.rb

```diff
@@ -12,7 +12,7 @@
 
 class Delayed::Job
   scope :pending, ->{ where("locked_at IS NULL AND attempts = 0") }
-  scope :awaiting_retry, ->{ where("failed_at IS NULL AND attempts > 0") }
+  scope :awaiting_retry, ->{ where("failed_at IS NULL AND attempts > 0 AND locked_at IS NULL") }
```

Comment:
I think this is a good change, but it will affect the number shown in the upper right for the number of jobs awaiting retry.  That number will no longer show jobs that are running right now.  What do you think of this change, @dsander?